### ISSUE TITLE
Scaffold errors in Routes.js

### ISSUE THREAD

**Issue by pi0neerpat** [redwoodjs/redwood#3056]
### Intended behavior

` yarn rw g scaffold Farm ` should add all relevant components, including components in `Routes.js`.

### What happens

Everything works, except updating `Routes.js`. Only the import `FarmsLayout` is added.

```
  System:
    OS: Linux 5.8 Ubuntu 20.04.1 LTS (Focal Fossa)
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 14.16.1 - /tmp/yarn--1626475999427-0.38991696470629567/node
    Yarn: 1.22.5 - /tmp/yarn--1626475999427-0.38991696470629567/yarn
  Browsers:
    Firefox: 84.0.2
  npmPackages:
    @redwoodjs/core: ^0.35.1 => 0.35.1
```

```bash
 yarn rw g scaffold Farm 
yarn run v1.22.5
$ /home/dev/repos/oneclickdapp-v2/node_modules/.bin/rw g scaffold Farm
  ✔ Generating scaffold files...
    ✔ Successfully wrote file `./web/src/components/Farm/EditFarmCell/EditFarmCell.js`
    ✔ Successfully wrote file `./web/src/components/Farm/Farm/Farm.js`
    ✔ Successfully wrote file `./web/src/components/Farm/FarmCell/FarmCell.js`
    ✔ Successfully wrote file `./web/src/components/Farm/FarmForm/FarmForm.js`
    ✔ Successfully wrote file `./web/src/components/Farm/Farms/Farms.js`
    ✔ Successfully wrote file `./web/src/components/Farm/FarmsCell/FarmsCell.js`
    ✔ Successfully wrote file `./web/src/components/Farm/NewFarm/NewFarm.js`
    ✔ Successfully wrote file `./api/src/graphql/farms.sdl.js`
    ✔ Successfully wrote file `./api/src/services/farms/farms.js`
    ✔ Successfully wrote file `./api/src/services/farms/farms.scenarios.js`
    ✔ Successfully wrote file `./api/src/services/farms/farms.test.js`
    ✔ Successfully wrote file `./web/src/layouts/FarmsLayout/FarmsLayout.js`
    ✔ Successfully wrote file `./web/src/pages/Farm/EditFarmPage/EditFarmPage.js`
    ✔ Successfully wrote file `./web/src/pages/Farm/FarmPage/FarmPage.js`
    ✔ Successfully wrote file `./web/src/pages/Farm/FarmsPage/FarmsPage.js`
    ✔ Successfully wrote file `./web/src/pages/Farm/NewFarmPage/NewFarmPage.js`
  ✔ Adding layout import...
  ✔ Adding set import...
  ✖ Adding scaffold routes...
    → object null is not iterable (cannot read property Symbol(Symbol.iterator))
    Adding scaffold asset imports...
object null is not iterable (cannot read property Symbol(Symbol.iterator))
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Code

```graphql
model Farm {
  id    String     @id @default(uuid())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  mnemonic String @unique
  contract String
  name String?
  description String?
  network String? @default("unknown")
  creator User @relation(fields: [creatorId], references: [id])
  creatorId  String
  impactText String
  backgroundColor String
  foregroundColor String
  stakingToken String
  rewardToken String
  rewardTokenName String
  stakingTokenName String
  iconImageSrc String
}
```

```bash
yarn rw g scaffold Farm
```

```js
// Routes.js after running scaffold
import { Set, Router, Route, Private } from '@redwoodjs/router'
import FarmsLayout from 'src/layouts/FarmsLayout'
import DefaultLayout from 'src/layouts/DefaultLayout'

const Routes = ({ useAuth }) => {
  return (
    <Router useAuth={useAuth}>
      <Route path="/" page={HomePage} name="home" />
      <Route path="/login" page={LoginPage} name="login" />
      <Route path="/dapps" page={DappsPage} name="dapps" />
      <Private unauthenticated="login">
        <Route path="/users/{id}/edit" page={EditUserPage} name="editUser" />
        <Route path="/users/{id}" page={UserPage} name="user" />
        <Route path="/new" page={NewDappPage} name="newDapp" whileLoadingAuth={() => 'Loading...'} />
        <Route path="/{mnemonic}/edit" page={EditDappPage} name="editDapp" />
      </Private>
      <Route path="/{mnemonic}" page={DappPage} name="dapp" />
      <Route path="/embed/{mnemonic}" name="embed" page={EmbedPage} />
      <Route notfound page={NotFoundPage} />
    </Router>
  )
}

export default Routes
```

**Comment by thedavidprice** [comment]
Definitely a problem @pi0neerpat!

The Scaffold generator regex was looking for `<Router>`, so `<Router useAuth={useAuth}>` is throwing it off.

Fix here #3084

**Comment by pi0neerpat** [comment]
Woot! :fire: :fire:

**Comment by thedavidprice** [comment]
released in v0.35.2 (in case you didn't see it)
